### PR TITLE
Zhongwen config updates

### LINKED ISSUE #5961: Zhongwen & Portuguese(Brasil) review
**Is your feature request related to a problem? Please describe.**
We want to release Zhongwen and Portuguese front pages. We should check all translations, configuration, and links. 

**Describe the solution you'd like**
- We should check requirements with Jon.
- We should check the service config as well as markup of the page to ensure we have translations for all offscreen text as well as on screen text.
- We should check Chartbeat config, ATI config, consent banners, metadata, navigation links, footer links, trust project links. Note that this should be done for these pages:

https://www.test.bbc.com/zhongwen/simp
https://www.test.bbc.com/zhongwen/trad
https://www.test.bbc.com/portuguese

- If there are any bugs, they should be fixed.
- Then we should review with UX designers (Laura and Ed).

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Unit tests should be sufficient, plus manual checks.

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #5961

**Overall change:** 
Updated translations and links for both simp and trad variants.

**Code changes:**
- Updated trustProject and footer links in both variants in `./lib/config/zhongwen`
- ~Updated `Contact the BBC` translation.~

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by sareh — APPROVED** [review]
👍 Looks good.

File: src/app/lib/config/services/zhongwen.js [hunk 1]
```diff
@@ -67,20 +69,20 @@ export const service = {
...
         },
         {
           href: 'https://www.bbc.com/usingthebbc/cookies/',
           text: 'Cookies',
         },
         {
-          href: 'https://www.bbc.com/contact/',
-          text: '联络BBC',
+          href: 'https://www.bbc.com/zhongwen/simp/institutional-38664417',
+          text: '与BBC联络',
```
Comment (jimjohnsonrollings) [inline src/app/lib/config/services/zhongwen.js]: This doesn't match the spreadsheet...

**Review by itimirichard — APPROVED** [review]

File: src/app/lib/config/services/zhongwen.js [hunk 2]
```diff
@@ -277,20 +281,20 @@ export const service = {
...
         },
         {
           href: 'https://www.bbc.com/usingthebbc/cookies/',
           text: 'Cookies',
         },
         {
-          href: 'https://www.bbc.com/contact/',
-          text: '聯絡BBC',
+          href: 'https://www.bbc.com/zhongwen/trad/institutional-38664417',
+          text: '與BBC聯絡',
```
Comment (jimjohnsonrollings) [inline src/app/lib/config/services/zhongwen.js]: Ditto

**Comment by hotinglok** [comment]
Moving back into Code Review until translations get confirmed.

**Comment by jimjohnsonrollings** [comment]
@lokh01 Confirming that we should use the translations as they were before :)

**Review by jimjohnsonrollings — APPROVED** [review]

**Pull request merged**